### PR TITLE
English language and xml file updates

### PR DESCRIPTION
**Pull request by rocco27**
- Changes in the english/resource files
------------------------------------------------------------------------
- Old Minecraft xml files removed, reason wrong suffixes
- Old Minecraft Bukkit xml files removed, reason wrong file names and suffixes
- Old Minecraft Tekkit xml files removed, reason wrong file names and suffixes
- Created a new and improved Minecraft xml files for all OS types
- Created a new and improved Bukkit xml files for all OS types
- Created a new and improved Tekkit xml files for all OS types
- Squad xml files updated
- Replaced the Minecraft icon to a new one which looks nicer
- Created a new icon for Bukkit (missed)
- Created a new icon for Feed the Beast (missed)
- Created a new icon for Spigot (missed)
- Created a new icon for Tekkit (missed)
------------------------------------------------------------------------
- Changes in Game Manager module for Rsync to use new game names for Bukkit and Tekkit
------------------------------------------------------------------------
To do in an another pull request to avoid GitHub conflict again:
- New Feed the Beast xml files update WIP and coming ASAP
- New Spigot xml files update WIP and coming ASAP

### CODE REVIEW

**Comment by rocco27** [comment]
I will add updated xml files for all other Minecraft clones probably tomorrow, so please do not merge yet if every changes are okey. ;)

**Comment by FarukGamer** [comment]
Do not forget Spigot, it's a Minecraft server aswell ;)

**Comment by rocco27** [comment]
If Spigot is a Java applications as Minecraft and other clones, then we can make Windows support too.

**Comment by FarukGamer** [comment]
Minecraft Server and their forks are Cross-platform compatible. Spigot is also a Java application.
Here a small details about the supported Minecraft xml's here

Minecraft (Vanilla) Server - basic unmodified vanilla gameserver

Bukkit - modificated version of Vanilla with support for Server side plugins and customizations

Spigot - A way more powerful and performance improved Fork of Bukkit, also with tons of customizations.

Feed the beast - A modified Fork of the vanilla server with Game mods capabilities. It uses the ForgeMod

**Comment by own3mall** [comment]
Is this ready to be merged yet?

**Comment by rocco27** [comment]
No yet, I still upload some more files asap.

**Comment by rocco27** [comment]
I checked the Spigot Windows version and I will create the xml file which is missing from OGP.

**Comment by rocco27** [comment]
I think it is ready right now, the second part (Feed the Beast and Spigot xml updates) are coming later.

**Comment by rocco27** [comment]
@own3mall It is ready to recheck and merge. I will countinue the work with Feed the Beast and Spigot xml files later. Now I just stopped, because I got GitHub conflict before, but I solved.

File: lang/English/modules/dashboard.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 
 define('dashboard_ftp_text', "The FTP client can compress files and send them via email, can uncompress too, edit text files and much more.");
-define('dashboard_addons_manager_text', "Manage all different types of services with OGP. Choose from over 70 servers supplied directly by OGP.");
+define('dashboard_addons_manager_text', "Manage all different types of services with OGP. Choose from over 115 servers, which supplied directly by OGP.");
```
Comment (Zorrototo) [inline lang/English/modules/dashboard.php]: previous sentence was correct, you just needed to change the number.

File: lang/English/modules/user_games.php [hunk 2]
```diff
@@ -192,7 +192,7 @@
 define('remove_it_anyway', "Would you like to remove it anyway?");
 define('sucessfully_deleted', "Sucessfully deleted %s");
 define('the_agent_had_a_problem_deleting', "The agent had a problem deleting %s, check the agent log");
-define('connection_timeout_or_problems_reaching_the_agent', "Connection timeout or problems reaching the agent");
+define('connection_timeout_or_problems_reaching_the_agent', "Connection timeout or problems with reaching the Agent");
```
Comment (Zorrototo) [inline lang/English/modules/user_games.php]: previous sentence was correct

**Pull request merged**